godtools/Views/TractElements/TractModal.swift

```diff
@@ -9,5 +9,101 @@
 import UIKit
 
 class TractModal: BaseTractElement {
+    
+    // MARK: Positions constants
+    
+    static let contentWidth: CGFloat = 275.0
+    
+    // MARK: - Positions and Sizes
+    
+    override func yEndPosition() -> CGFloat {
+        return self.height
+    }
+    
+    // MARK: - Object properties
+    
+    var properties = TractModalProperties()
+    var modalOpen = false
+    
+    // MARK: - Setup
+    
+    override func setupView(properties: [String: Any]) {
+        loadElementProperties(properties: properties)
+        TractBindings.addBindings(self)
+        
+        self.frame = buildFrame()
+        setupStyle()
+    }
+    
+    func setupStyle() {
+        self.backgroundColor = UIColor.black.withAlphaComponent(0.8)
```

Comment:
Two thoughts:
1. You could probably use .black.withAlphaComponent(0.8)
2. Will the modals always be black, or are they subject to styling through the manifest defaults? /cc @frett